shahar's-PRs/challenge-#6/launch-EC2.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/bash
+# This script built to launch an instance in aws
+# VPC
+echo "creating VPC"
+aws ec2 create-vpc --tag-specifications 'ResourceType=vpc,Tags=[{Key=Name,Value=Bakery_VPC}]' --cidr-block 10.0.0.0/16
+VPC_ID=`aws ec2 describe-vpcs --filter Name=tag:Name,Values=Bakery_VPC --query Vpcs[].VpcId --output text`
```

Comment:
While it's works it better to use $() 
http://mywiki.wooledge.org/BashFAQ/082

Reply:
Fixed :)